src/Main.java

```diff
@@ -60,13 +40,39 @@ public static void main(String[] args) {
         });
 
         System.out.println("You have reached your user limit.");
-        printUserList(userNames, userAges);
+        printUserList(users);
         System.exit(0);
-            }
+    }
 
-    private static void printUserList(List<String> userNames, List<Integer> userAges) {
-        IntStream.range(0, userNames.size())
-                .forEach(i -> System.out.println("User" + (i + 1) + ": " + userNames.get(i) + ", " + userAges.get(i)));
+    private static void printUserList(List<User> users) {
+        IntStream.range(0, users.size())
```

Comment:
You can use user.stram() instead of this  IntStream.range(0, users.size())

Reply:
complete